spring-cloud-kubernetes-fabric8-discovery/src/main/java/org/springframework/cloud/kubernetes/fabric8/discovery/EndpointSubsetNS.java

```diff
@@ -24,10 +24,4 @@
  * @author Haytham Mohamed
  **/
 record EndpointSubsetNS(String namespace, List<EndpointSubset> endpointSubset) {
-	EndpointSubsetNS {
```

Comment:
I don't know why we were passing `null` and creating an empty List, we could do that at the point of were we called this constructor...